test/libsolidity/SolidityExecutionFramework.h

```diff
@@ -72,8 +83,7 @@ class SolidityExecutionFramework: public dev::test::ExecutionFramework
 		}
 		eth::LinkerObject obj = m_compiler.object(_contractName.empty() ? m_compiler.lastContractName() : _contractName);
 		BOOST_REQUIRE(obj.linkReferences.empty());
-		sendMessage(obj.bytecode + _arguments, true, _value);
-		return m_output;
+		return std::move(obj.bytecode);
```

Comment:
Oh well, this cannot use move since it is still available after as `m_compiler.object`.

Reply:
Hm, then again `.object` is a reference, but we make a copy as `obj`, so we could do move.

@chriseth which one do you prefer: `const& obj` or move?